src/core_plugins/choropleth/public/choropleth_controller.js

```diff
@@ -0,0 +1,152 @@
+import uiModules from 'ui/modules';
+import 'plugins/kbn_vislib_vis_types/controls/vislib_basic_options';
+import _ from 'lodash';
+import AggConfigResult from 'ui/vis/agg_config_result';
+import KibanaMap from 'ui/vis_maps/kibana_map';
+import FilterBarFilterBarClickHandlerProvider from 'ui/filter_bar/filter_bar_click_handler';
```

Comment:
i think you might not actually need to import this, it should be available on vis.listeners.click